ta/pkcs11/src/attributes.c

```diff
@@ -192,6 +192,44 @@ enum pkcs11_rc get_attribute(struct obj_attrs *head, uint32_t attribute,
 	return PKCS11_CKR_OK;
 }
 
+enum pkcs11_rc set_attribute(struct obj_attrs **head, uint32_t attribute,
```

Comment:
Should this behave in a away that if attribute was not there then it gets added, if it was there then it gets replaced?